FAQ.md

```diff
@@ -151,6 +151,21 @@ mapping:
 /network-interfaces/0 -> eth1
 ```
 
+### How can I gracefully reboot the guest? How can I gracefully poweroff the guest?
```

Comment:
`mdl` is reporting the following error:
FAQ.md:154: MD026 Trailing punctuation in header

Otherwise LGTM.

Reply:
What does that mean though?

Reply:
Looking at https://github.com/markdownlint/markdownlint/issues/72 it seems like `?` is by default disallowed punctuation in headers.

I believe for the FAQ it is totally normal to have headers as Frequently Asked **Questions**, so I propose we ignore the error.

Reply:
Ok.